cmd/controller-manager/app/insecure_serving.go

```diff
@@ -41,6 +43,10 @@ func (s *InsecureServingInfo) Serve(handler http.Handler, shutdownTimeout time.D
 		MaxHeaderBytes: 1 << 20,
 	}
 
-	glog.Infof("Serving insecurely on %s", s.Listener.Addr())
+	if len(s.Name) > 0 {
+		glog.Infof("Serving %s insecurely on %s", s.Name, s.Listener.Addr())
+	} else {
+		glog.Infof("Serving insecurely on %s", s.Listener.Addr())
+	}
```

Comment:
could be re-structured as:
```
name := s.Name
if len(name) > 0 {
	name += " "
}
glog.Infof("Serving %sinsecurely on %s", name, s.Listener.Addr())
```
which would end storing a single literal in the read-only pool.